flow-typed/box-ui-elements.js

```diff
@@ -512,6 +512,11 @@ type Collaborators = {
     entries: Array<SelectorItem>
 };
 
+type Translations = {
+    translationEnabled?: boolean,
+    onTranslate?: Function
```

Comment:
Does this function need to return anything?

Reply:
Nope it sets the `translatedTaggedMessage` prop in Comment.js